public/scripts/parseSaves.js

```diff
@@ -136,7 +136,7 @@ exports.parseToText = function (saveFile, _callback) {
         })
     });
 
-    saveWriter.close();
+    saveWriter.end();
```

Comment:
```suggestion
    saveWriter.end(function() {
    console.log('File Write Completed');
    _callback();
    });
```
Then remove the .on('finish') function. :)
(I hope this is what I'm supposed to do)